EIPS/eip-5192.md

```diff
@@ -0,0 +1,50 @@
+---
+eip: 5192
+title: Minimal Soulbound NFTs
+description: Minimal standard interface for soulbinding EIP-721 NFTs
+author: Tim Daubenschütz (@TimDaub)
+discussions-to: https://ethereum-magicians.org/t/eip-5192-minimal-soulbound-nfts/9814
+status: Review
+type: Standards Track
+category: ERC
+created: 2022-07-01
+requires: 165, 721
+---
+
+## Abstract
+
+This standard is an extension of [EIP-721](./eip-721.md). It proposes a minimal interface to make tokens soulbound using the feature detection functionality of [EIP-165](./eip-165.md).
+
+## Motivation
+
+The Ethereum community has expressed a need for non-transferrable, non-fungible, and socially-priced tokens similar to World of Warcraft’s soulbound items. But the lack of a token standard leads many developers to simply throw errors upon a user's invocation of transfer functionalities. Over the long term, this will lead to fragmentation and less composability.
+
+In this document, we outline a minimal addition to [EIP-721](./eip-721.md) that allows wallet implementers to check for a token contract's permanent (non-)transferrability using [EIP-165](./eip-165.md).
+
+## Specification
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY" and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+### Contract Interface
+
+Upon minting an EIP-5192 token, it must be permanently inseparable from the receiving account. All [EIP-721](./eip-721.md) functions of the contract that transfer the token from one account to another must throw.
+
+To aid recognition that an [EIP-721](./eip-721.md) token implements soulbinding via `EIP-5192`, upon calling [EIP-721](./eip-721.md)'s `function supportsInterface(bytes4 interfaceID) external view returns (bool)` with EIP-5192's `SOULBOUND_VALUE=0x9e7d7f8` as `bytes4 interfaceID`, a contract implementing `EIP-5192` must return `true`.
```

Comment:
I think perhaps something even simpler (and much more versatile!) is to add a `function soulbound(uint256 tokenId) external view returns (bool tokenIdSoulbound)`, which would even allow for soulbounding to be managed on a per-tokenId basis. This is by no means necessary though.

Reply:
This can be known implicitly by calling `ownerOf(tokenId)` in the [4973 spec](https://eips.ethereum.org/EIPS/eip-4973)

Reply:
> I like the interface you proposed, a token's "soulboundness" can be known implicitly by calling `ownerOf(tokenId)` in the [4973 spec](https://eips.ethereum.org/EIPS/eip-4973).

no. Unless I'm missing something